src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import duke.commands.Command;
+import java.util.Scanner;

```

Comment:
Wrong Import statement order
```suggestion
import java.util.Scanner;

import duke.commands.Command;
```